README.md

```diff
@@ -54,7 +54,7 @@ jobs:
       uses: sonarsource/sonarqube-scan-action@v2.0.2
       env:
         SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-        SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}
+        SONAR_HOST_URL: <Sonar host URL value>
```

Comment:
In the How section of the ticket we say: `Update documentation to use env variables instead of secrets`.
I don't think that is what we are doing here. I would expect to have something like `${{ vars.SONAR_HOST_URL}}` now

Reply:
And that make me things that we would also need to update the onboarding tutorials accordingly I guess.

Reply:
Yepp good call:
GHA has the guide about vars here: https://docs.github.com/en/actions/learn-github-actions/variables#using-the-vars-context-to-access-configuration-variable-values